src/hooks/useHideBodyScroll.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react'
+
+export const useHideBodyScroll = (isOpen: boolean): null => {
+  useEffect(() => {
+    if (isOpen) {
+      document.body.style.overflow = 'hidden'
+    } else {
+      const currentOverflow = document.body.style.overflow
+      if (currentOverflow === 'hidden') {
```

Comment:
I don't understand this check. Do you need to assign `document.body.style.overflow` to 'hidden' when its value already 'hidden'?

Reply:
I need to set it initial, when sidesheet i sclosed